modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -244,4 +244,4 @@ Configuration properties can be overwritten. The Mule runtime engine uses the fo
 
 So, for example, if a configuration property named `size` is defined as a system property, and there is also an application configuration property named `size`, the value for the application is the value of the property with the most precedence (in this case, the system property).
 
-Also, a property can derive its value from other properties that are higher in the hierarchy. Therefore, an application property can derive its value from environment, system, and/or deployment properties. A system property can derive its value from a deployment property, and so on. For example, if there is a system property named `env`, an application configuration property can have the value `file.$&#123;env&#125;.xml`.
+Also, a property can derive its value from other properties that are higher in the hierarchy. Therefore, an application property can derive its value from environment, system, and/or deployment properties. A system property can derive its value from a deployment property, and so on. For example, if there is a system property named `env`, an application configuration property can have the value `file.$&#123;env&#125;.xml`. On the other hand, an Application Property cannot depend on a Application property's value unless it's defined in the same file. For example, a Secure Configuration Property cannot depend on a Configuration Property. 
```

Comment:
On the other hand, an Application Property cannot depend on a Application property's value > On the other hand, an application property cannot depend on an application property's value

Reply:
```suggestion
Also, a property can derive its value from other properties that are higher in the hierarchy. Therefore, an application property can derive its value from environment, system, and/or deployment properties. A system property can derive its value from a deployment property, and so on. For example, if there is a system property named `env`, an application configuration property can have the value `file.$&#123;env&#125;.xml`.  On the other hand, an application property cannot depend on an application property's value unless it's defined in the same file. For example, a Secure Configuration Property cannot depend on a Configuration Property. 
```